Kingsley_ansible_playbook.yml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Configure TodoApp Environment
+  hosts: webservers
+  become: yes
+  remote_user: ec2-user
+ 
+  vars_prompt:
+    - name: trello_api_key
+      prompt: "Enter Trello API Key"
+      private: yes
+      
+    - name: trello_api_token
+      prompt: "Enter Trello API Token"
+      private: yes
+
+    - name: board_id
+      prompt: "Enter Trello Board ID"
+      private: no
+      
+    - name: to_do_list_id
+      prompt: "Enter To Do List ID"
+      private: no
+
+    - name: done_list_id
+      prompt: "Enter Done List ID"
+      private: no
+
+  tasks:
+  
+    - name: Install git using Yum
+      ansible.builtin.yum:
+        name: git
+        state: present
+      become: yes
+    
+    - name: Install latest version of Python
+      ansible.builtin.yum:
+        name: python
+        state: latest
+      become: yes
+ 
+    - name: Install Poetry using curl if not installed
+      ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+      args: 
+        creates: /home/ec2-user/.local/bin/poetry
+          
+    - name: Create /opt/todoapp directory
+      ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        owner: ec2-user
+        group: ec2-user
+        mode: '0755'
+      become_user: root
+     
+    - name: Clone Git repository
+      ansible.builtin.git:
+        repo: 'https://github.com/kingbrookey/DevOps-Course-Starter.git'
+        dest: /opt/todoapp
+        version: module_3
+        force: yes
+      become: yes
+      become_user: root
+
+    - name: Copy pyproject.toml to managed server
+      ansible.builtin.copy:
+        src: /home/ec2-user/pyproject.toml
+        dest: /opt/todoapp/pyproject.toml 
+
+    - name: Install project dependencies using Poetry
+      ansible.builtin.command: ~/.local/bin/poetry install
+      become: yes
+      become_user: ec2-user
```

Comment:
I think this is a sign you don't need to `become` at all, since `ec2-user` is your default user anyway